spyder/plugins/ipythonconsole/scripts/spyder-kernels.bat

```diff
@@ -0,0 +1,13 @@
+:: This scripts helps activate a conda environment before running a spyder-kernel
```

Comment:
Please rename this script to `conda-activate.bat`

Reply:
Done